### DISCUSSION TITLE
Who Is Doing Each Feature?

### DISCUSSION THREAD

**Discussion by srothetafesa** [srothetafesa/Tafe24S1T109_Ext#2] [General]
Hi John,

We need to decide which feature we would each like to implement into the project. The features we need to implement are a mortgage calculator and a currency converter. Please let me know your preference.

Cheers,
Sam

**Comment by JohnLSison** [comment]
Hey Sam, 

I'll be doing the currency converter and will name it feature2 branch.

Cheers,
John
